# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../heimdal/loadedclass.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- heimdal-0.6-20031010/lib/hdb/hdb-ldap.c.orig	2003-10-30 04:12:47.000000000 +0000
+++ heimdal-0.6-20031010/lib/hdb/hdb-ldap.c	2003-10-30 04:13:35.000000000 +0000
@@ -310,6 +310,17 @@
 	if (ret != 0) {
 	    goto out;
 	}
+	/* inetOrgPerson is the 2nd structural object class */
+	ret = LDAP_addmod(&mods, LDAP_MOD_ADD, "objectClass", "inetOrgPerson");
+	if (ret != 0) {
+	    goto out;
+	}
+	/* hordePerson is the 3rd structural object class */
+	ret = LDAP_addmod(&mods, LDAP_MOD_ADD, "objectClass", "hordePerson");
+	if (ret != 0) {
+	    goto out;
+	}
+	/* sambaSamAccount should go here when SID generation is figured out */
 	ret =
 	    LDAP_addmod(&mods, LDAP_MOD_ADD, "objectClass",
 			"krb5Principal");
